data/component-design-tokens/text-area-design-tokens.json

```diff
@@ -1,65 +1,79 @@
 {
   "text-area-border": {
-    "value": "inset 0 0 0 {borderWidth.s} {color.greyscale.700}",
+    "value": "inset 0 0 0 {inputs.border-width.default} {inputs.color.border.default}",
```

Comment:
Remove the `s` from `inputs` to match your changes from your [Text Input](https://github.com/GovAlta/design-tokens/pull/91/files#diff-8c78087aa7cd5e1b8b8a1dd731e75d799d438cd973caa5f139424963887a60f0) PR.